addons/packages/calico/3.19.1/bundle/config/overlay/update_overlay.yaml

```diff
@@ -0,0 +1,31 @@
+#@ load("@ytt:overlay", "overlay")
+#@ load("@ytt:data", "data")
+
+#@overlay/match by=overlay.subset({"kind":"DaemonSet"}), expects="0+"
+---
+spec:
+  #@overlay/match missing_ok=True
+  updateStrategy:
+    #@overlay/match missing_ok=True
+    type: OnDelete
+
+#@overlay/match by=overlay.subset({"kind":"Deployment"}), expects="0+"
+---
+spec:
+  #@overlay/match missing_ok=True
+  strategy:
+    #@overlay/match missing_ok=True
+    type: RollingUpdate
```

Comment:
For calico-kube-controllers, the default strategy is Recreate. Because controllers can only have one single active instance, otherwise there will be potential problem like reconciling at the same time and creating duplicate resources.

For more details see here: https://github.com/projectcalico/calico/blob/master/getting-started/kubernetes/installation/calico-kube-controllers.yaml#L19-L25